### PR TITLE
Yan/set data ip - support data ips for inaugurator

### PR DESCRIPTION
**Pull request by yant-lb**
Added some changes to support data ip configuration in the inaugurator for osmosis usage

### CODE REVIEW

File: inaugurator/network.py [hunk 1]
```diff
@@ -43,6 +48,14 @@ class NetworkInterface:
...
         self.iface = iface_name
 
+    def get_current_ip_with_subnet(self):
+        cmd = "/usr/sbin/ip -4 addr show dev %s | grep inet | tr -s " " | cut -d" " -f3 | head -n 1" % self.iface
+        sh.run(cmd)
+
+    def set_ip(self, new_ip_with_subnet):
+        cmd = "/usr/sbin/ifconfig %s %s" % (self.iface, new_ip_with_subnet)
+        sh.run(cmd)
+
```
Comment (amittar-lb) [inline inaugurator/network.py:58]: where do we use these functions?

File: inaugurator/ceremony.py [hunk 2]
```diff
@@ -229,26 +254,28 @@ def _osmosFromNetwork(self, destination, timeout_after=20*60): #20min
...
             message = self._talkToServer.label()
-            self._label = json.loads(message)['rootfs']
+            self._handle_label_message(message)
         else:
             self._label = self._args.inauguratorNetworkLabel
         ATTEMPTS = 2
         signal.signal(signal.SIGALRM, self._raise_timeout_exception)
         signal.alarm(timeout_after)
+        osmosis_object_stores = self._osmosis_data_ip_port if self._osmosis_data_ip_port \
+            else self._args.inauguratorOsmosisObjectStores
```
[the remark this answers is not in the document]
Comment (yant-lb) [reply inaugurator/ceremony.py:264]: Done.

File: inaugurator/network.py [hunk 3]
```diff
@@ -9,14 +9,19 @@ class Network:
     _CONFIG_SCRIPT_PATH = "/etc/udhcp_script.sh"
     _NR_PING_ATTEMPTS = 40
 
-    def __init__(self, macAddress, ipAddress, netmask, gateway):
+    def __init__(self, macAddress, ipAddress, netmask=None, gateway=None):
         self._gateway = gateway
         interfacesTable = self._interfacesTable()
         assert macAddress.lower() in interfacesTable, "macAddress %s interfacesTable %s" % (macAddress, interfacesTable)
         interfaceName = interfacesTable[macAddress.lower()]
         sh.run("/usr/sbin/ifconfig lo 127.0.0.1")
```
[the remark this answers is not in the document]
Comment (yant-lb) [reply inaugurator/network.py:17]: Done.

File: inaugurator/network.py (hunk 1 above)
Comment (yant-lb) [reply inaugurator/network.py]: removed

**Review by amittar-lb — APPROVED** [review]

**Pull request merged**